src/test/java/org/reaktivity/nukleus/oauth/internal/control/ControllerIT.java

```diff
@@ -93,92 +95,95 @@ public void shouldFailToResolveWithTooManyRoles() throws Exception
     })
     public void shouldFailToResolveWithTooManyRealms() throws Exception
     {
+        thrown.expect(either(is(instanceOf(IllegalStateException.class)))
+                .or(is(instanceOf(ExecutionException.class))));
+        thrown.expectCause(either(nullValue(Exception.class)).or(is(instanceOf(IllegalStateException.class))));
+
```

Comment:
Is it sufficient to remove these `thrown` lines and change annotation to `@Test( exception = IllegalStateException.class)` instead?
If so, that would be preferred as it is simpler.

Reply:
I just tried it and it should work as `expected = ExecutionException.class`. I don't think this specific test will run into an `IllegalStateException` so I can go ahead and change that.